server/bin/docker/standalone_test.json

```diff
@@ -173,7 +173,7 @@
               "downsampleAggregator": "max",
               "downsampleFillPolicy": "none",
               "downsampleInterval": "1s",
-              "metric": "sys.cpu0.user",
+              "metric": "sys.cpu.user",
```

Comment:
After I got the docker bits working I noticed that sys.cpu0.user wasn't being inserted by timely.util.InsertTestData so I just changed all of the standalone_test.json sys.cpu0.user to sys.cpu.user where I found it under the assumption they were both intended to be used together.

I didn't change the volume use in the docker-compose so it still uses grafana/timely-app/src/datasource/dashboards/standalone_test.json.